core/src/com/goonsquad/galactictd/systems/graphics/RenderSystem.java

```diff
@@ -38,10 +37,20 @@
     private Camera camera;
     private Texture missingTextureImage;
 
+    private float drawX;
+    private float drawY;
+    private float drawOriginX;
+    private float drawOriginY;
+    private float drawWidth;
+    private float drawHeight;
+    private float drawRotation;
+    private boolean drawCurrentEntity;
+
     public RenderSystem(Camera camera, Aspect.Builder aspect, Texture missingTextureImage) {
         super(aspect.all(Layer.class).one(Text.class, Renderable.class));
```

Comment:
Should we consider adding Spatial to this aspect? All of these entities seem to require it to be drawn, and I think they all have it already, but it seems like it should be here.